dts/arm/st/u0/stm32u0.dtsi

```diff
@@ -273,6 +307,115 @@
 			status = "disabled";
 		};
 
+		dma1: dma@40020000 {
+			compatible = "st,stm32-dma-v2";
+			#dma-cells = <3>;
+			reg = <0x40020000 0x400>;
+			interrupts = <9 0 10 0 11 0>;
+			clocks = <&rcc STM32_CLOCK(AHB1, 0U)>;
+			dma-channels = <7>;
+			dma-offset = <0>;
+			dma-requests= <12>;
+			status = "disabled";
+		};
+
+		dma2: dma@40020400 {
+			compatible = "st,stm32-dma-v2";
+			#dma-cells = <3>;
+			reg = <0x40020400 0x400>;
+			interrupts = <11 0>;
+			clocks = <&rcc STM32_CLOCK(AHB1, 1U)>;
+			dma-channels = <5>;
+			dma-offset = <7>;
+			dma-requests= <12>;
+			status = "disabled";
+		};
+
+		dmamux1: dmamux@40020800 {
+			compatible = "st,stm32-dmamux";
+			#dma-cells = <3>;
+			reg = <0x40020800 0x400>;
+			interrupts = <11 0>;
+			dma-channels = <12>;
+			dma-generators = <4>;
+			dma-requests= <76>;
+			status = "disabled";
+		};
+
+		spi1: spi@40013000 {
+			compatible = "st,stm32-spi-fifo", "st,stm32-spi";
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <0x40013000 0x400>;
+			clocks = <&rcc STM32_CLOCK(APB1_2, 12U)>;
+			resets = <&rctl STM32_RESET(APB1H, 12U)>;
+			interrupts = <25 0>;
+			status = "disabled";
+		};
+
+		spi2: spi@40003800 {
+			compatible = "st,stm32-spi-fifo", "st,stm32-spi";
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <0x40003800 0x400>;
+			clocks = <&rcc STM32_CLOCK(APB1, 14U)>;
+			resets = <&rctl STM32_RESET(APB1L, 14U)>;
+			interrupts = <26 0>;
+			status = "disabled";
+		};
+
+		spi3: spi@40003c00 {
+			compatible = "st,stm32-spi-fifo", "st,stm32-spi";
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <0x40003c00 0x400>;
+			clocks = <&rcc STM32_CLOCK(APB1, 15U)>;
+			resets = <&rctl STM32_RESET(APB1L, 15U)>;
+			interrupts = <26 0>;
+			status = "disabled";
+		};
+
+		usb: usb@40005c00 {
+			compatible = "st,stm32-usb";
+			reg = <0x40005c00 0x400>;
+			interrupts = <8 0>;
+			interrupt-names = "usb";
+			num-bidir-endpoints = <8>;
+			ram-size = <2048>;
```

Comment:
According to the refman RM0503, Size of dedicated packet buffer memory SRAM for USB should be `1024` bytes instead of 2048